test/EndToEnd/GivenDotNetUsesMSBuild.cs

```diff
@@ -19,7 +19,7 @@ public void ItCanNewRestoreBuildRunCleanMSBuildProject()
             {
                 string projectDirectory = directory.Path;
 
-                string newArgs = "console --debug:ephemeral-hive";
+                string newArgs = "console -f netcoreapp1.1 --debug:ephemeral-hive";
```

Comment:
Yep, this is the new default, but given that it was quite confusing trying to track down some of the issues that we saw in getting our PR to go green, I think it's best to be explicit here (as it's much harder to search for the absence of something to fix up the next time the default changes)